packages/suite-native/tsconfig.json

```diff
@@ -270,6 +270,7 @@
         "./src/App.tsx",
         "./index.d.ts",
         "./src/utils/suite/oauth.ts", // todo: remove when used somewhere, otherwise eslint complaints
+        "../suite/global.d.ts",
```

Comment:
nope, react-native env should not have `Window` declared
i'm guessing that you've added it bc. you are getting errors after using `window.desktopApi` in middleware

middlewares and actions in general shouldn't have `window` references otherwise you need to recreate them in suite-native src structure (like routerActions)

we should reduce using `window` to the necessary minimum (components, utils)